src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1015,4 +1015,7 @@ I agree to all of the foregoing:</value>
   <data name="Decompiler_Legal_Notice_Title" xml:space="preserve">
     <value>Decompiler Legal Notice</value>
   </data>
-</root>
+  <data name="Code_style_header_use_editor_config" xml:space="preserve">
+    <value>The settings configured here only apply to your machine. To configure these settings to travel with your solution, use EditorConfig.</value>
```

Comment:
".editorconfig files"? (with the dot). People who know what the file is might understand this better then, as right now it's less obvious that's the mechanism being used here.

Reply:
are you talking about the text? I will wait @kuhlenh to comment on this.

Reply:
Good call Jason. I think that sounds better.

Reply:
done